src/target/stm32l4.c

```diff
@@ -851,7 +866,7 @@ static bool stm32l4_cmd_option(target_s *t, int argc, const char **argv)
 		tc_printf(t, "STM32L5 options not implemented!\n");
 		return false;
 	}
-	if (t->part_id == ID_STM32WBXX) {
+	if ((t->part_id == ID_STM32WBXX) || (t->part_id == ID_STM32WB1X)) {
```

Comment:
Drop the extra inner parentheses, they suppress a GCC warning we want enabled here to detect accidental assignment in the if statement.